### ISSUE TITLE
Tensorflow Pretrained Model

### ISSUE THREAD

**Issue by dipampatel18** [vietnguyen1991/AirGesture#2]
Can you tell me exactly which pretrained model did you use for the project?

Also, how many different gestures can it recognize other than closed and open?

Thanks!

**Comment by vietnh1009** [comment]
The pretrained model provided by tensorflow, which I put at src/pretrained_model.pb. It could be used to detect hand(s), with only 2 gestures as you mentioned

**Comment by dipampatel18** [comment]
Yes I saw that. I meant to ask what is the name of the model that you downloaded or where exactly did you get it from? The reason is I want to look into the documentation of that model and have an in-depth idea about it.

**Comment by kingaza** [comment]
amazing~
@vietnguyen1991 could you also share your model? thx a lot.

**Comment by QiangZiBro** [comment]
Same, this model runs fast in cpu, how did you design and train the model?@uvipen
/
